src/Doctrine/ORM/CountOrderItemUnitRefundsBelongingToOrderQuery.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Doctrine\ORM;
+
+use Sylius\Component\Core\Repository\OrderItemUnitRepositoryInterface;
+
+class CountOrderItemUnitRefundsBelongingToOrderQuery implements CountRefundsBelongingToOrderQueryInterface
```

Comment:
as query services with one method only, we could finalize it. There is nothing we can change in this class
```suggestion
final class CountOrderItemUnitRefundsBelongingToOrderQuery implements CountRefundsBelongingToOrderQueryInterface
```